### PR TITLE
Maxwell

### PR DESCRIPTION
**Pull request by Maxwell-Tian**
modified

### CODE REVIEW

**Comment by oldeight888** [comment]
Good job!

File: src/main/java/api/API_request.java [hunk 1]
```diff
@@ -52,7 +53,11 @@ public API_request(List<Ingredient> ingredients, String health, String cuisine,
     }
 
     public List<List<String>> Searching_Recipe(){
-        return Recipe_info_analyzer(Requesting());
+        String feedback = Requesting();
```
Comment (Maxwell-Tian) [inline src/main/java/api/API_request.java:56]: no prob

**Review by Maxwell-Tian — COMMENTED** [review]
good to go

**Review by Maxwell-Tian — COMMENTED** [review]
good

**Review by oldeight888 — APPROVED** [review]

**Review by LucyK222 — APPROVED** [review]

**Comment by Icarus02423** [comment]
Well done! Our API is almostlt done!

**Pull request closed without merging**